exercises/03_count_byte/main_test.go

```diff
@@ -0,0 +1,25 @@
+package main
+
+import "testing"
+
+func TestCountByte(t *testing.T) {
+	for _, tc := range []struct {
+		name   string
+		input  string
+		input2 byte
+		want   int
+	}{
+		{"letter is not repeated", "mango", byte('l'), 0},
```

Comment:
nit: You don't have to cast `'l'` to byte. `'l'` is an untyped constant, and Go will automatically make a byte out of it.